assets/model_monitoring/components/generation_safety_quality/generation_safety_quality_signal_monitor/spec.yaml

```diff
@@ -138,13 +140,14 @@ jobs:
       type: aml_token
   output_signal_metrics: 
     type: spark
-    component: azureml://registries/azureml/components/model_monitor_metric_outputter/versions/0.3.4
+    component: azureml://registries/azureml/components/model_monitor_metric_outputter/versions/0.3.5
     inputs:
       signal_metrics:
         type: mltable
         path: ${{parent.jobs.compute_metrics.outputs.signal_metrics}}
       signal_type: "GenerationSafetyQuality"
       signal_name: ${{parent.inputs.signal_name}}
+      metric_timestamp: ${{parent.inputs.monitor_current_time}}
```

Comment:
Missing param - monitor_name here. I think this got trickled down to service also

Reply:
Actually service is fine

Reply:
Added it, thank you!